docs/development/deploying-locally/index.md

```diff
@@ -18,62 +19,79 @@ $ source ~/.virtualenvs/reana/bin/activate
+| `--component` / `-c`       | Which components to clone (shortname, name, `.`, `CLUSTER`, `ALL`)                 | CLUSTER                   |
+| `--user` / `-u`            | GitHub user name.                                                                  | anonymous                 |
+
+**3.** Create a new REANA cluster:
 
 ```console
-(reana) $ make setup prefetch
+(reana) $ reana-dev cluster-create
 ```
 
+| Option                     | Description                                                                        | Default value             |
+|----------------------------|------------------------------------------------------------------------------------|---------------------------|
+| `--mount` / `-m`           | Which local directories to mount in the cluster (e.g. `-m /var/reana:/var/reana`)  | -                         |
+| `--mode`                   | Which mode to run REANA cluster (`releasehelm`, `releasepypi`,`latest`, `debug`)   | latest                    |
+
 **4.** Build all components:
 
 ```console
-(reana) $ make build
+(reana) $ reana-dev cluster-build
 ```
 
+| Option                     | Description                                                                        | Default value             |
+|----------------------------|------------------------------------------------------------------------------------|---------------------------|
+| `--build-arg` / `-b`       | Any build arguments (e.g. `-b COMPUTE_BACKENDS=kubernetes,htcondorcern,slurmcern`) | -                         |
+| `--exclude-components`     | Which components to exclude from build.                                            | -                         |
+| `--mode`                   | Which mode to run REANA cluster (`releasehelm`, `releasepypi`,`latest`, `debug`)   | latest                    |
+| `--no-cache`               | Do not use Docker image layer cache.                                               | false                     |
+
 **5.** Deploy REANA:
 
 ```console
-(reana) $ make deploy
+(reana) $ reana-dev cluster-deploy
 ```
 
+| Option                     | Description                                                                        | Default value             |
+|----------------------------|------------------------------------------------------------------------------------|---------------------------|
+| `--job-mounts` / `-j`      | Which directories from the Kubernetes nodes to mount inside the job pods.          | -                         |
+| `--namespace`              | Kubernetes namespace.                                                              | default                   |
+| `--mode`                   | Which mode to run REANA cluster (`releasehelm`, `releasepypi`,`latest`, `debug`)   | latest                    |
+| `--values` / `-v`          | Which Helm configuration values file to use.                                       | values-dev.yaml           |
+
```

Comment:
Could you please add the new options to pass admin email, password and instance name introduced [here](https://github.com/reanahub/reana/pull/391/files#diff-be62d283d45d14a354849d15e8b46c66R244)?

Reply:
Yes! Make sure to propagate feedback over them to this PR, just in case you end up changing the option name.